src/platforms/react-native/migration.mdx

```diff
@@ -1,8 +1,39 @@
 ---
-title: Migrating from react-native-sentry
+title: Migration
 ---
 
-If you are upgrading from an earlier version of react-native-sentry you should unlink the package to ensure the generated code is updated to the latest version:
+## From 1.x to 2.x
+
+In `@sentry/react-native` version `2.x`, release health and native stack traces are now enabled by default.
```

Comment:
I think we should also mention the envelope endpoint, see https://docs.sentry.io/platforms/apple/guides/ios/migration/#store-endpoint.

Reply:
Good point

Reply:
Good catch. Will do.